moved check for townhall location

janikz211 opened the pull request:
this change is to avoid unnecessary time searching for pumps and drills when TH sniping

i also find the droplspell() on this file unnecessary as zapping dead bases atm is not efficient (not included on this change)

janikz211 committed:
moved check for townhall location

this change is to avoid unnecessary time searching for pumps and drills when TH sniping

i also find the droplspell() on this file unnecessary as zapping dead bases atm is not efficient (not included on this change)

kaganus commented:
If ($chkRedArea) Then
If $bBtnAttackNowPressed = True Then 

you are also skipping these checks. doesn't th snipe use red area for deployment?
also if the user pressed attack now, it will not attack and leave if you move it up

janikz211 committed:
Update algorithm_AllTroops.au3

reverted back to original position and just added $opttrophymode check to skip checking red area for TH snipe

janikz211 commented:
nop... th snipe doesnt use red area for depolyment afaik
reverted the position back to original and added an $opttrophymode check to skip checking red area for TH snipe...please do correct me if i did the coding wrong.. :D
